generate_fixtures.py

```diff
@@ -186,11 +187,51 @@ def __init__(self):
         self.write_and_publish_repository()
 
 
+class TUFTestFixtureThresholdTwo(TUFTestFixtureBase):
+    def __init__(self):
+        super().__init__()
+        (public_timestamp_key_2, private_timestamp_key_2) = self.write_and_import_keypair(
+            'timestamp_2')
+        self.repository.timestamp.add_verification_key(public_timestamp_key_2)
+        self.repository.timestamp.load_signing_key(private_timestamp_key_2)
+        self.repository.timestamp.threshold = 2
+        self.repository.mark_dirty(['timestamp'])
+        self.write_and_publish_repository(export_client=True)
+
+
+class TUFTestFixtureThresholdTwoAttack(TUFTestFixtureBase):
```

Comment:
is this supposed to extend TUFTestFixtureThresholdTwo ?

Reply:
I think extended didn't work as desired - instead I copied some of the code below

Reply:
> is this supposed to extend TUFTestFixtureThresholdTwo ?

Sharp eye. I'll fix this.

Reply:
I switched it to extend the other class. The output looks correct to me.